apps/modernization-ui/src/design-system/richTooltip/rich-tooltip.module.scss

```diff
@@ -0,0 +1,14 @@
+.richtooltipcontainer {
+  position: absolute;
+  margin: 0;
+  padding: 16px;
+  font-size: 14px;
+  line-height: 20px;
+  background-color: white;
+  color: black;
+  outline-color: #E6E7E8;
+  outline-width: 1px;
+  outline-style: solid;
+  filter: drop-shadow(0px 4px 2px #9f9f9f);
```

Comment:
Question: Are these colors defined in the `styles/_colors.scss`? If not, might want to confirm with designer about using one that is already defined.

Reply:
forgot to fix this, thanks!